frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -76,6 +77,36 @@ const forestClientApi = apiServiceFactory.getForestClientApi();
 const idirBceidProxyApi = apiServiceFactory.getIdirBceidProxyApi();
 const userRoleAssignmentApi = apiServiceFactory.getUserRoleAssignmentApi();
 
+const initialSteps = ref({
+    userInfo: {
+        stepNumber: 1,
+        label: 'User Information',
+        active: false,
+        icon: IconSteps.queue,
+        errorMessage:''
+    },
+    addUser: {
+        stepNumber: 2,
+        label: 'Add User Roles',
+        active: false,
+        icon: IconSteps.queue,
+        errorMessage: ''
+    },
+});
+
+const orgInfo = ref({
+    label: 'Organization Information',
+    active: false,
+    isVisible: false,
+    icon: IconSteps.queue,
+    errorMessage: ''
+});
+
+stepItems.value = [
```

Comment:
Hi Jota, we don't update state variable like this. Let's create a method in the store/stepState.ts file to handle the value change, something like the following and then import it to use:
```
export const setStepItems= (newStepItemsValue) => {
    stepItems.value = newStepItemsValue;
}
```
And if you're trying to initialize the stepItem, you should do in the onMounted method, or when you want to assign the value, it shouldn't be here.

Reply:
Hello! Sure, will do!